### DISCUSSION TITLE
索引建好了，但是搜索时分页跳转失败，也就是每次只能看到1-3页

### DISCUSSION THREAD

**Discussion by 18935568736** [AlistGo/alist#4781] [Invalid question]
因为网盘文件比较多，建索引很长时间，最后文件大小800m
现在的问题是，索引构建成功，但是搜索出来的记录页面跳转存在问题。比如搜索记录48页，主界面是第1页，第2页，第3页、下一页（>)箭头和48页，但是现在选择第二页和第三页，搜索记录会正常变动，但是最下面分页界面还是第一页标粗了；点击下一页(>)和48页每次还是显示第3页的数据，最下面分页页面还是第一页标粗。我现在只能查询到前3页的数据，求助这个搜索分页问题怎么解决？

**Comment by haoduck** [comment]
的确，是有这个问题，下一页没有正确跳转
